srcopsmetrics/visualization.py

```diff
@@ -472,7 +472,7 @@ def _visualize_top_x_issues_types_wrt_developers(
         )
         return fig
 
-    def _visualize_top_X_issues_types_wrt_project(
```

Comment:
Is this function not used anywhere?

Reply:
visualization functions will probably not be used furthermore, as we decided to move onto the Superset dashboards

therefore, as i mentioned in PR description, I suggest removing all the unused code & errors issued by `pre-commit` in other PR (in order for this PR to be more clean and specific)

any thoughts on this @pacospace ?

Reply:
the 'refactor request issue' here #169

Reply:
I think make sense to remove what we do not need anymore, although in this case someone could use the visualizations locally maybe using the one available

Reply:
yes, but you can also use Superset locally, right?

Reply:
Superset locally?